src/scenes/world-map.js

```diff
@@ -72,6 +72,15 @@ export class WorldMapScene extends Phaser.Scene {
                 y: MAP.AL_KHARID_FURNACE.Y,
                 key: CONSTANTS.SCENES.AL_KHARID_FURNACE,
             },
+            {
+                text: "Varrock Anvil",
+                condition: characterData.getQuestCompleted(
+                    CONSTANTS.PREREQUISITES.VARROCK_ANVIL
```

Comment:
For whatever reason this never resolves to true and I can't get in the anvil level. It's possible al kharid is never set to true for some reason

Reply:
Did you mean you can't go into the "Varrock Anvil" level after making 15 bronze bars? I'm not having any issues on my end.

Reply:
Oh I didn't realize there was a number of bars minimum